templates/terraform/examples/cloud_identity_group_membership.tf.erb

```diff
@@ -0,0 +1,42 @@
+resource "google_cloud_identity_group" "group" {
+  provider = google-beta
+  display_name = "<%= ctx[:vars]['id_group'] %>"
+
+  parent = "customers/<%= ctx[:test_env_vars]['cust_id'] %>"
+
+  group_key {
+  	id = "<%= ctx[:vars]['id_group'] %>@<%= ctx[:test_env_vars]['org_domain'] %>"
+  }
+
+  labels = {
+    "cloudidentity.googleapis.com/groups.discussion_forum" = ""
+  }
+}
+
+resource "google_cloud_identity_group" "child-group" {
+  provider = google-beta
+  display_name = "<%= ctx[:vars]['id_group'] %>-child"
+
+  parent = "customers/<%= ctx[:test_env_vars]['cust_id'] %>"
+
+  group_key {
+  	id = "<%= ctx[:vars]['id_group'] %>-child@<%= ctx[:test_env_vars]['org_domain'] %>"
+  }
+
+  labels = {
+    "cloudidentity.googleapis.com/groups.discussion_forum" = ""
+  }
+}
+
+resource "google_cloud_identity_group_membership" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  group    = google_cloud_identity_group.group.name
```

Comment:
Can this be `.id` instead of `.name`? https://github.com/terraform-providers/terraform-provider-google/wiki/Developer-Best-Practices#self-links-names-and-ids